_pytest/test_config.py

```diff
@@ -0,0 +1,74 @@
+from rasa_nlu.config import RasaNLUConfig
+import json
+import os
+
+
+defaults = {
+  "backend": "mitie",
+  "config": "config.json",
+  "data": None,
+  "emulate": None,
+  "language": "en",
+  "mitie_file": "./data/total_word_feature_extractor.dat",
+  "path": os.getcwd(),
```

Comment:
in tests it's defined as ./data, so this key of the dict will be different during tests